src/main/kotlin/no/nav/amt/deltaker/bff/application/plugins/Routing.kt

```diff
@@ -1,28 +1,44 @@
 package no.nav.amt.deltaker.bff.application.plugins
 
 import io.ktor.http.HttpStatusCode
+import io.ktor.serialization.jackson.jackson
 import io.ktor.server.application.Application
 import io.ktor.server.application.call
 import io.ktor.server.application.install
+import io.ktor.server.plugins.contentnegotiation.ContentNegotiation
 import io.ktor.server.plugins.statuspages.StatusPages
 import io.ktor.server.response.respondText
 import io.ktor.server.routing.get
 import io.ktor.server.routing.routing
 import no.nav.amt.deltaker.bff.application.registerHealthApi
 import no.nav.amt.deltaker.bff.auth.AuthorizationException
+import no.nav.amt.deltaker.bff.auth.TilgangskontrollService
+import no.nav.amt.deltaker.bff.deltaker.DeltakerService
+import no.nav.amt.deltaker.bff.deltaker.api.registerDeltakerApi
 
-fun Application.configureRouting() {
+fun Application.configureRouting(
+    tilgangskontrollService: TilgangskontrollService,
+    deltakerService: DeltakerService,
+) {
     install(StatusPages) {
         exception<AuthorizationException> { call, cause ->
             call.respondText(text = "403: $cause", status = HttpStatusCode.Forbidden)
         }
+        exception<NoSuchElementException> { call, cause ->
+            call.respondText(text = "404: $cause", status = HttpStatusCode.NotFound)
+        }
         exception<Throwable> { call, cause ->
             call.respondText(text = "500: $cause", status = HttpStatusCode.InternalServerError)
         }
     }
+    install(ContentNegotiation) {
```

Comment:
Jeg vet ikke om jeg skjønner helt men, har vi ikke allerede installert `ContentNegotiation` i `Serialization.kt`?

Reply:
Vi har installert det for klienten, men ikke for server-delen 😅